config/routes.rb

```diff
@@ -280,6 +280,9 @@
     get "/rails/mailers/*path" => "rails/mailers#preview"
   end
 
+  get '/quickin'     => 'users#quickin'
```

Comment:
These two routes should also be wrapped in an `if Rails.env.development?` statement.

Reply:
This is a good point. I had never considered whether Rails.env could be used in routes but its ruby so certainly should add.